KatelynKim/writegether
# chore: remove MUI and all related code

## Description

Now that we've decided to just stick with TailwindCss, remove the MUI packages and all related code.
This ticket also extends the existing `tailwind.config.ts` to include our custom color palette.

Closes #28 
 
## How Has This Been Tested?

1. Check out this branch.
2. Run `npm i` to install the packages
3. Run `npm run dev` and ensure the app runs successfully.
4. Carefully review all source code to see if there's any MUI-related code left.
